### PR TITLE
Impletement Internationalisation for Home component

### PR DESCRIPTION
**Pull request by KnyazNiyaz**

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -37,9 +45,18 @@ import 'rxjs/add/observable/of';
...
+        useFactory: (createTranslateLoader),
+        deps: [HttpClient]
+      }
+    }),
+    HttpModule,
+    HttpClientModule,
     BrowserModule,
     FormsModule,
-    HttpModule,
+    HttpClientModule,
```
Comment (ashish173) [inline src/app/app.module.ts]: @KnyazNiyaz do we have `HttpClientModule` twice ?

Comment (KnyazNiyaz) [reply src/app/app.module.ts]: @ashish173 Removed

**Comment by ashish173** [comment]
@KnyazNiyaz we'll take some to review your PR as it's a big one. @mpvineesh you'll have to wait for this one to get merged. cc @pkrawat1 @kartikjagdale @zeus999

File: src/assets/i18n/de.json [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+		"BROWSE": "Browse",
+		"ALL_CATEGORIES":"All Categories",
+		"TRENDING_PROJECT": "Trending Projects",
+		"VIEW_ALL": "View All",
+		"VIDEO_INTRODUCTION": "Play Video for Our Introduction",
+		"CUSTOMER_FEEDBACK": "What Our Customer Say’s"
+	},
+	"LOGIN_DIALOG": {
+		"LOGIN": "Login",
+		"SIGNUP_TXT": "New to Crowdpouch Sign up Now!",
```
Comment (ashish173) [inline src/assets/i18n/de.json]: @KnyazNiyaz I think you have used the files from the crowd funding project. We need to have those keys only which are used here in this project. Please make sure you make this change before this PR can be merged. cc: @zeus999

**Comment by ashish173** [comment]
@KnyazNiyaz I see a major flaw in this PR, there are unnecessary keys from the crowd funding project present in the files I would request you to remove them and update the PR.

File: src/assets/i18n/de.json (hunk 2 above)
Comment (KnyazNiyaz) [reply src/assets/i18n/de.json]: that's better?

**Comment by ashish173** [comment]
@KnyazNiyaz could you please add the files for other languages with the same keys as in english translation file? Once that's done we'll be able to merge the PR.

File: src/app/home/index.ts [hunk 3]
```diff
@@ -45,6 +47,7 @@ import { FilterPipe } from './content/product-list/product-filter.pipe';
   imports: [
     RouterModule.forChild(routes),
     SharedModule,
+    TranslateModule
```
Comment (laertispappas) [inline src/app/home/index.ts]: Shouldn't this be imported and exported in `SharedModule` ?

**Comment by pkrawat1** [comment]
Closing this PR as we it is not and cannot be synced with current module structure.
Thanks @KnyazNiyaz for the PR.

**Pull request closed without merging**